src/components/CompareResults/beta/RevisionSelect.tsx

```diff
@@ -41,6 +55,8 @@ function RevisionSelect() {
             </Box>
           );
         }}
+        onChange={(event) => handlerChangeComparison(event.target.value)}
+        value={activeComparison}
       >
         {revisionsOptions.map((option) => (
           <MenuItem key={option} value={option}>
```

Comment:
(For the follow-up to use a key for "All Revisions", this is where the key could be different than the value)

Reply:
[PCF-287 Change string to key for 'All revisions' option](https://mozilla-hub.atlassian.net/browse/PCF-287)